client/src/components/UIUXUtils.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const SVG = styled.svg`
+  stroke: #bdbdbd;
+  stroke-width: 3;
+  stroke-linecap: square;
+  stroke-linejoin: square;
+  fill: none;
+  position: absolute;
+  right: 12px;
+  top: calc(50% - 6.5px);
+  ${props => props.isActive ? 'transform: rotate( 180deg );' : ''}
+  transition: transform 0.2s ease-in-out;
+`;
+
+// Mobile side menu down/up arrow
+export const ChevronDownIcon = ({ isActive }) => (
+  <SVG width='20' height='13' isActive={isActive}>
+    <polyline points='3 3, 10 10, 17 3'></polyline>
+  </SVG>
+);
+
+ChevronDownIcon.propTypes = {
+  isActive: PropTypes.bool
+};
+
+// Steam / Valve logos - different styling for mobile menu vs. desktop menu
+export const Logo = styled.img`
+  cursor: ${props => props.isFooter ? 'default' : 'pointer'};
+  padding-top: ${props => props.isMobile ? '0' : '30px'};
+  margin-right: ${props => props.isMobile ? '0' : '25px'};
+  width: ${props => props.isMobile ? 'auto' : (props.isFooter ? '92px' : '176px')};
```

Comment:
Piggybacking on Leslie's comment about nested ternaries here.
I'm sure there are many opinions on solutions to this. Here's one I found that looks pretty clean:

https://github.com/styled-components/styled-components/issues/129#issuecomment-255541233

Reply:
This tip was incredibly helpful! I ended up using this for my nested ternaries for both services. Thanks!